diff --git a/include/OsqpEigen/Solver.hpp b/include/OsqpEigen/Solver.hpp
index 2e77651..c5d8dea 100644
--- a/include/OsqpEigen/Solver.hpp
+++ b/include/OsqpEigen/Solver.hpp
@@ -88,6 +88,10 @@ namespace OsqpEigen
          */
                   ~Solver();

+        Solver(Solver&&) = default;
+
+        Solver& operator=(Solver&&) = default;
+
         /**
                   * Initialize the solver with the actual initial data and settings.
                             * @return true/false in case of success/failure.
